intelmq/bots/experts/aggregate/expert.py

```diff
@@ -46,8 +51,14 @@ def cleanup(self):
                     event.add("extra.count", int(data['c']))
                     event.add("extra.time_end", data['l'])
                     self.send_message(event)
+                    counter_sent += 1
+                else:
+                    counter_ignored += 1
                 self.cache_get_redis_instance().delete(key)
             self.__next_cleanup = int(time.time()) + 10
+            self.logger.debug('Completed Cleanup. Messages sent: %d, messages ignored: %d.', counter_sent, counter_ignored)
+        else:
+            self.logger.debug('Skipped Cleanup (%fs < 10s).', self.__next_cleanup - time.time())
```

Comment:
This might get spammy in the logs, because each event received will run the cleanup process.

Reply:
OTOH it's the debug log, which logs *at least* one line for every incoming message anyway